### PR TITLE
[VL] Add centos8 on Velox CI

### PR DESCRIPTION
**Pull request by zhejiangxiaomai**
## What changes were proposed in this pull request?

This patch adds one more GHA job on CentOS8/Ubuntu22.04. The job will do packaging and TPC-H/DS test

## How was this patch tested?

GHA passing

### CODE REVIEW

File: .github/workflows/velox_be.yml [hunk 1]
```diff
@@ -168,3 +168,51 @@ jobs:
...
+          ./build_arrow_for_velox.sh && \
+          cd /opt/gluten/ep/build-velox/src
+          ./get_velox.sh && \
+          ./build_velox.sh'
+      - name: Build Gluten CPP library
+        run: |
+          docker exec velox-backend-centos8-test-$GITHUB_RUN_ID bash -c '
+          source /opt/rh/gcc-toolset-9/enable && \
+          cd /opt/gluten/cpp && \
+          mkdir build && cd build && cmake .. && make -j'
```
Comment (jinchengchenghh) [inline .github/workflows/velox_be.yml:201]: Now we repeatedly build protobuf in CI, both build velox and build cpp library, maybe we can set `BUILD_PROTOBUF` default to `OFF`, https://github.com/oap-project/gluten/blob/main/cpp/CMakeLists.txt#L45

Comment (zhejiangxiaomai) [reply .github/workflows/velox_be.yml:201]: I will check it.

File: tools/gluten-te/centos/defaults.conf [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+# JVM jdwp bind port
+DEFAULT_JDWP_SERVER_PORT=5005
+
+# Docker options
+DEFAULT_EXTRA_DOCKER_OPTIONS="--network bridge"
+
+# Build will result in this image
+DEFAULT_DOCKER_TARGET_IMAGE_TPC=gluten-te/centos/gluten-tpc
+DEFAULT_DOCKER_TARGET_IMAGE_TPC_GDB=gluten-te/centos/gluten-tpc-gdb
+DEFAULT_DOCKER_TARGET_IMAGE_TPC_GDB_SERVER=gluten-te/centos/gluten-tpc-gdb-server
```
Comment (zhztheplayer) [inline tools/gluten-te/centos/defaults.conf:71]: Can we remove the unused options? Only some of them are useful for cbash.sh

Comment (zhejiangxiaomai) [reply tools/gluten-te/centos/defaults.conf:71]: Yes, I will remove it.

**Review by zhouyuan — APPROVED** [review]
👍

**Pull request merged**